### PR TITLE
feat(perf): Add HTTP response code status definitions

### PR DESCRIPTION
**Pull request by gggritso**
A bit of sparkle, so we can remind people of what the stranger response codes mean when they debug network requests.

### CODE REVIEW

File: static/app/views/performance/http/definitions.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  '207': t('Multi-Status'),
+  '208': t('Already Reported'),
+  '226': t('IM Used'),
+  '300': t('Multiple Choices'),
+  '301': t('Moved Permanently'),
+  '302': t('Found'),
+  '303': t('See Other'),
+  '304': t('Not Modified'),
+  '305': t('Use Proxy'),
+  '306': t('unused'),
```
Comment (wmak) [inline static/app/views/performance/http/definitions.tsx:25]: 🤔 is this intentionally lower cased?
```suggestion
  '306': t('Unused'),
```

**Review by wmak — APPROVED** [review]
oooh nice

File: static/app/views/performance/http/definitions.tsx (hunk 1 above)
Comment (gggritso) [reply static/app/views/performance/http/definitions.tsx]: Oh, well spotted, I'm gonna remove this, it's literally _not in use_

Comment (wmak) [reply static/app/views/performance/http/definitions.tsx]: 😆

**Review by AbhiPrasad — APPROVED** [review]
I wonder if we should link to MDN docs as well

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/100

Would actually be nice to have all around the product 🤔

**Pull request merged**